posts/2023-07-24-leadership-council-update.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "July 2023 Leadership Council Update"
+author: Leadership Council
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+
+Hello again from the Rust Leadership Council. In our [first blog post][fr1st post], we laid out several immediate goals for the council and promised to report back on their progress. It has been about a month since our first update so we wanted to share how it's going and what we're working on now.
+
+[fr1st post]: https://blog.rust-lang.org/2023/06/20/introducing-leadership-council.html
+
+## Accomplishments so far
+
+In our first post, we wrote:
+
+> In the coming weeks we will be establishing the basic infrastructure for the group, including creating a plan for regular meetings and a process for raising agenda items, setting up a team repository, and ultimately completing the transition from the former Rust leadership structures.
+
+We have made progress on each of these things.
+
+For the last month we have been holding weekly meetings and will continue to do so for at least the next month. In the long term, we would like these meetings to become less frequent and rely more on public, asynchronous discussion and decision processes, but in the short term there has been plenty to discuss each week. We publish minutes from each meeting, and these are linked at the [bottom of this post](#meeting-minutes).
+
+We have created committees for two of our most urgent needs: one to create a process for electing Project Directors to serve as our representatives on the foundation board, and one of council librarians to work on processes to document our actions and decisions.
```

Comment:
It is probably helpful to outline what a "committee" is here.

Reply:
Also, generally elaborating on this paragraph would be helpful, rather than a half-sentence description of each committee.

Reply:
Something like:

> In order to be able to make progress without needing to involve all Council members, we've decided to establish small groups of people (called "committees") to focus on various specific work items. The two committees we've established so far are...